specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-09-01/storagecache.json

```diff
@@ -1274,6 +1274,16 @@
                   "x-ms-identifiers": [
                     "name"
                   ]
+                },
+                "logSpecifications": {
```

Comment:
>logSpecifications [](http://example.com/codeflow?start=17&length=17)

Pl add it in a new API version

Reply:
Hi @rkmanda: It is already added in our new version, we are adding this in our old versions because we want to document it as S360 is flagging us. This property is a part of the GET response of a metadata API which has no impact on the customers.